assets/src/scss/styleguide.scss

```diff
@@ -177,17 +177,17 @@ code {
 // p.typeSpecimen::after  { content: "Font stack: #{ko-font(base)}"; }
 
 .sg-typeSpecimen--body {
-	font-size: ko-rem(30);
+	font-size: 30px;
```

Comment:
@ashleynolan as I mentioned in [], all font-based values that are written in `px` will be converted to rems at build stage by the PostCSS-pxtorem function.

The reason is that all conversion functions written in Sass are extremely fragile, particularly when there's an unknown type used as the argument. The pxtorem process is extremely reliable and the new dev experience is much more easy to understand - **just write in px and we'll do the rest.** 🙂